src/services/metadata/MetadataService.ts

```diff
@@ -193,7 +193,7 @@ export class MetadataService extends MarketplaceApi {
    * @param did - DID of the asset.
    * @returns DDO of the asset.
    */
-  private async retrieveDDO(did?: DID | string, metadataServiceEndpoint?: string): Promise<DDO> {
+  public async retrieveDDO(did?: DID | string, metadataServiceEndpoint?: string): Promise<DDO> {
```

Comment:
we had set this to private to prevent skipping over the resolution policy

Reply:
I think is not necessary that protection because as said above there are scenarios where we don't have on-chain connection but we know in which marketplace api is the asset metadata stored